package.json

```diff
@@ -51,8 +52,13 @@
   },
   "dependencies": {},
   "devDependencies": {
```

Comment:
were going to want to remove object.entries and object.values since they should by added via babel

Reply:
Not quite sure which part you are talking.

Reply:
I see.